types/http-proxy/index.d.ts

```diff
@@ -161,14 +164,23 @@ declare class Server extends events.EventEmitter {
   listenerCount(type: string): number;
 }
 
+interface ParsedUrl {
```

Comment:
Can't we use the real `Url` interface from the node `url`?
```typescript
import { Url } from "url";
export type ParsedUrl = Url;
```

You have a type `ProxyTargetUrl` which already do this on `line 18`.